analyzer/src/main/kotlin/managers/GoDep.kt

```diff
@@ -73,6 +73,9 @@ private val GO_LEGACY_MANIFESTS = mapOf(
 
 /**
  * The [Dep](https://golang.github.io/dep/) package manager for Go.
+ *
+ * Note: Dep was an official experiment to implement a package manager for Go. As of 2020, Dep is deprecated and
+ * archived in favor of Go modules, which have had official support since Go 1.11.
```

Comment:
Should we say "Go **M**odules" (capitalized first) for consistency, even tough this is a modification of this copied text?